picasso/src/main/java/com/squareup/picasso/Request.java

```diff
@@ -434,6 +434,21 @@ public Builder transform(Transformation transformation) {
       return this;
     }
 
+    /**
+     * Add a list of custom transformations to be applied to the image.
+     * <p>
+     * Custom transformations will always be run after the built-in transformations.
+     */
+    public Builder transform(List<? extends Transformation> transformations) {
```

Comment:
Sorry - I added the cast because I took your advice and tried it with `<? super Transformation>` first.  The test I added didn't compile, so I changed it to `<? extends Transformation>`.  So I just removed the unnecessary cast and it should be good now.